README.md

```diff
@@ -57,6 +57,7 @@ Property | Meaning
 
 ### Example Configuration
 ```yaml
+proxyListeningPort: 80 #the port the proxy will listen on
```

Comment:
The example isn't the right place for documentation using a comment. Please remove the comment part, I will add some documentation after merging it.